src/Dropdown.tsx

```diff
@@ -136,22 +135,17 @@ const DropdownItem = styled.li`
   ${sx};
 `
 
-DropdownMenu.defaultProps = {
-  direction: 'sw',
-  theme
-}
+DropdownMenu.defaultProps = {direction: 'sw'}
 DropdownMenu.displayName = 'Dropdown.Menu'
 
-DropdownItem.defaultProps = {theme}
 DropdownItem.displayName = 'Dropdown.Item'
 
-DropdownButton.defaultProps = {theme, ...Button.defaultProps}
+DropdownButton.defaultProps = {...Button.defaultProps}
```

Comment:
Does this still need to be spread if we aren't adding anything to it or is it bad form to directly assign another component's default props?

Reply:
I was curious about that too. I think it's probably fine. I'll update it.